[2023-04-26T04:22:19.099+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-04-26T04:22:17.671249+00:00 [queued]>
[2023-04-26T04:22:19.107+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-04-26T04:22:17.671249+00:00 [queued]>
[2023-04-26T04:22:19.107+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T04:22:19.108+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T04:22:19.108+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T04:22:19.118+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-04-26 04:22:17.671249+00:00
[2023-04-26T04:22:19.124+0000] {standard_task_runner.py:55} INFO - Started process 21629 to run task
[2023-04-26T04:22:19.129+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-04-26T04:22:17.671249+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpj7eki6n9']
[2023-04-26T04:22:19.131+0000] {standard_task_runner.py:83} INFO - Job 139: Subtask create_table
[2023-04-26T04:22:19.189+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-04-26T04:22:17.671249+00:00 [running]> on host bebb68623a17
[2023-04-26T04:22:19.240+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T04:22:17.671249+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T04:22:17.671249+00:00
[2023-04-26T04:22:19.249+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-04-26T04:22:19.254+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
                first_name varchar(255) not null,
                last_name varchar(255) not null,
                country varchar(255) not null,
                username varchar(255) not null,
                password varchar(255) not null,
                email varchar(255) not null
            );
        , parameters: None
[2023-04-26T04:22:19.256+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-04-26T04:22:19.264+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230426T042217, start_date=20230426T042219, end_date=20230426T042219
[2023-04-26T04:22:19.302+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T04:22:19.329+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
